client/state/stats/lists/utils.js

```diff
@@ -172,6 +172,33 @@ export function parseOrderDeltas( payload ) {
 	} );
 }
 
+/**
+ * Create the correct property and value for a label to be used in a chart
+ *
+ * @param {string} unit - day, week, month, year
+ * @param {object} date - moment object
+ * @param {object} localizedDate - moment object
+ * @return {object}
+ */
+function getChartLabels( unit = '', date, localizedDate ) {
+	if ( ! date.isValid() || ! localizedDate.isValid() ) {
```

Comment:
i suppose you could use [isMoment](https://momentjs.com/docs/#/query/is-a-moment/) here to verify a moment object was passed in before calling moment methods on `date` || `localizedDate`